src/components/structures/TimelinePanel.js

```diff
@@ -732,7 +732,8 @@ var TimelinePanel = React.createClass({
         const events = this._timelineWindow.getEvents();
 
         // first find where the current RM is
-        for (var i = 0; i < events.length; i++) {
+        let i;
```

Comment:
it bothers me slightly that we use `i` outside of the loop, but not enough to warrant a refactor. Just documenting my concern.

Reply:
Yes, it is a bit unusual and surprising, I agree 😖